notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -73,6 +73,27 @@
         "id": "k-82uiXlTjvw"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        BUCKET_URI = f"gs://{BUCKET_NAME}"
</pre><p>Move this line outside the if statement.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>